scripts/prepare-release.sh

```diff
@@ -70,6 +70,8 @@ sed -E s",^([[:space:]]+)image:.+$,\1image: $container_image," -i *yaml.template
 
 # Patch Helm chart
 sed -e s"/appVersion:.*/appVersion: $release/" -i deployment/node-feature-discovery/Chart.yaml
+sed -e s"/pullPolicy: Always$/pullPolicy: IfNotPresent/" -i deployment/node-feature-discovery/values.yaml
```

Comment:
nit: you could combine all three rules into a single sed call, especially as you already use `-e`. Like this:
```
sed -e <rule1> \
    -e <rule-2> \
    -i <file>
```

nit: it would be more fool-proof to use wildcards similar to appVersion, i.e. `s"/pullPolicy:.*$/pullPolicy: IfNotPresent/"`

Reply:
```suggest
sed -e s"/pullPolicy:.*/pullPolicy: IfNotPresent/" \
```